src/diffusers/models/transformers/auraflow_transformer_2d.py

```diff
@@ -347,8 +347,9 @@ def __init__(
     def attn_processors(self) -> Dict[str, AttentionProcessor]:
         r"""
         Returns:
-            `dict` of attention processors: A dictionary containing all attention processors used in the model with
-            indexed by its weight name.
+            [`dict[`attention processors`]`]
```

Comment:
This would cause tests to fail too as it requires all implementation that copy from `UNet2DConditionalModel.attn_processors` to have the same docs and implementation. This can be changed, but you need to make the modification in the mentioned file and then run `make fix-copies` (can do in a separate PR)